src/carts/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "devDependencies": {
     "eslint": "3.19.0",
-    "eslint-plugin-header": "1.2.0",
+    "eslint-plugin-header": "git+https://github.com/herzog31/eslint-plugin-header.git",
```

Comment:
Will you create an issue to change this when your PR is merged?

Reply:
I created one already.